server/lib/get_user_database.js

```diff
@@ -0,0 +1,23 @@
+const pgClient = require('./pg_client');
+
+
+const getUserDatabase = (connection, userId, callback) => {
+  const client = pgClient(connection);
+  client.query(
+      `SELECT * FROM users
+       WHERE id = $1`,
+      [userId],
+    (postgresErr, result) => {
+      if (postgresErr) {
+        callback(postgresErr);
```

Comment:
you pass the error  to the callback, but its not handled there. At least in the testdb.js. I guess the test would fail anyways if there is an error, but when the function is called in production it should be handled in that callback. Just a note.